components/invitation_modal/invitation_modal_members_step.jsx

```diff
@@ -198,32 +203,33 @@ class InvitationModalMembersStep extends React.Component {
                         />
                     </h2>
                     <div data-testid='inputPlaceholder'>
-                        <FormattedMessage
-                            id='invitation_modal.members.search-and-add.placeholder'
-                            defaultMessage='Add members or email addresses'
-                        >
-                            {(placeholder) => (
-                                <UsersEmailsInput
-                                    usersLoader={this.usersLoader}
-                                    placeholder={placeholder}
-                                    ariaLabel={localizeMessage('invitation_modal.members.search_and_add.title', 'Invite People')}
-                                    onChange={this.onChange}
-                                    value={this.state.usersAndEmails}
-                                    validAddressMessageId={t('invitation_modal.members.users_emails_input.valid_email')}
-                                    validAddressMessageDefault='Invite **{email}** as a team member'
-                                    noMatchMessageId={t('invitation_modal.members.users_emails_input.no_user_found_matching')}
-                                    noMatchMessageDefault='No one found matching **{text}**, type email to invite'
-                                    onInputChange={this.onUsersInputChange}
-                                    inputValue={this.state.usersInputValue}
-                                />
-                            )}
-                        </FormattedMessage>
+                        <UsersEmailsInput
+                            usersLoader={this.usersLoader}
+                            placeholder={placeholder}
+                            ariaLabel={localizeMessage('invitation_modal.members.search_and_add.title', 'Invite People')}
+                            onChange={this.onChange}
+                            value={this.state.usersAndEmails}
+                            validAddressMessageId={t('invitation_modal.members.users_emails_input.valid_email')}
+                            validAddressMessageDefault='Invite **{email}** as a team member'
+                            noMatchMessageId={t('invitation_modal.members.users_emails_input.no_user_found_matching')}
```

Comment:
I think this string needs to be changed too, and probably we need to change the `showAddEmail` method in the UsersEmailsInput to never allow to add emails as valid options.

Reply:
Yeah, I've only fixed the cases mentioned in the ticket but you are right. We might need some extra input from the UX/PM side.

The problem is that we changed the placeholder and the help text but the user can still type in an email address and also if no user is found the error message will say:  
 `No one found matching **{text}**, type email to invite`

@michaelgamble or @wiersgallak , thoughts?

Reply:
@michaelgamble I am thinking of just removing the text "type email to invite" and retain "No one found matching **{text}**."  Please let me know if you have any other ideas.

Reply:
@wiersgallak I am guessing these modifications should apply to the guest invitation modal as well since we allow inviting existing guests to a channel even when email invitations are disabled. Correct?

Reply:
Yes I would modify for the guest invitation flow as well when email invitations are disabled.  @michaelgamble  was going to do a quick review as well.